### PR TITLE
Added -A flag to ssh connection

### PR DESCRIPTION
**Pull request by BradErz**
Fixes https://github.com/kubernetes/kops/issues/2530

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2531)

### CODE REVIEW

**Comment by BradErz** [comment]
You can verify the CLA now 👍

File: cmd/kops/update_cluster.go [hunk 1]
```diff
@@ -282,11 +282,11 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 			fmt.Fprintf(sb, " * validate cluster: kops validate cluster\n")
 			fmt.Fprintf(sb, " * list nodes: kubectl get nodes --show-labels\n")
 			if !usesBastion(instanceGroups) {
-				fmt.Fprintf(sb, " * ssh to the master: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+				fmt.Fprintf(sb, " * ssh to the master: ssh -A -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
```
Comment (chrislovecnm) [inline cmd/kops/update_cluster.go]: Is this the only place we print this? Also, we have another issue open about the user vs the os of the nodes.  Up to you if you want to fix it here as well.

Comment (kris-nova) [inline cmd/kops/update_cluster.go]: Also you only need (and probably should) forward your agent if you are using a bastion.

Comment (BradErz) [inline cmd/kops/update_cluster.go]: @kris-nova yea good point, I fixed that. 

@chrislovecnm can you link me here? i will have a look tonight

**Comment by BradErz** [comment]
CLA come back to me 😂

**Comment by chrislovecnm** [comment]
@k8s-bot ok to test

**Pull request merged**